src/style-auro-calendar-cell.scss

```diff
@@ -54,6 +60,7 @@
   &:hover {
     background-color: #f0f0f0; // var(--auro-color-ui-bkg-hover-on-light); - this token was part of the design but caused a problem with the `inRange` date styles so we had to switch to a solid color for hover.
     color: var(--auro-color-text-primary-on-light);
+
```

Comment:
I am not seeing a fix for the calendar gradient listed in ticket #141

Reply:
That part is not getting handled in this PR. There is a full review of the mobile that is going to happen.

Reply:
Right, work moved to this issue,  see https://github.com/AlaskaAirlines/auro-datepicker/issues/141#issuecomment-1702114839

- #140